langchain4j-vertex-ai-gemini/src/main/java/dev/langchain4j/model/vertexai/StreamingChatResponseBuilder.java

```diff
@@ -1,18 +1,22 @@
 package dev.langchain4j.model.vertexai;
 
-import com.google.cloud.vertexai.api.Candidate;
-import com.google.cloud.vertexai.api.GenerateContentResponse;
-import com.google.cloud.vertexai.generativeai.preview.ResponseHandler;
+import com.google.cloud.vertexai.api.*;
+import com.google.cloud.vertexai.generativeai.ResponseHandler;
 import dev.langchain4j.data.message.AiMessage;
 import dev.langchain4j.model.output.FinishReason;
 import dev.langchain4j.model.output.Response;
 import dev.langchain4j.model.output.TokenUsage;
 
 import java.util.List;
+import java.util.Optional;
 
 class StreamingChatResponseBuilder {
 
     private final StringBuffer contentBuilder = new StringBuffer();
+
+    private boolean hasFunctionCall = false;
```

Comment:
minor: do we really need this `boolean` field? Can we use `functionCall` null/not null?

Reply:
We can, I'll change that.

Reply:
Covered in 424b280991f53480981607c2695196ed50956738